lib/Conversion/TTIRToTTIRDecomposition/TTIRToTTIRDecomposition.cpp

```diff
@@ -146,46 +147,46 @@ generateTranspose(Value input, int64_t dim0, int64_t dim1,
       dim1_attr, operandConstraints);
 }
 
-static std::vector<TransposeDims> generateKernelTransposeIndices(
-    ttir::ConvolutionOp op,
-    const std::vector<int64_t> ttnn_convolution_kernel_layout) {
+static std::vector<TransposeDims>
+generateTransposeIndicesLeftToRight(std::vector<int64_t> current_layout,
```

Comment:
All these transpose methods are hard to follow. A comment (with an example) above each method would make it much easier to come back to.

Additionally, it's somewhat noisy to keep these method in this file, can they be moved to a util file of sorts?

Reply:
I've cleaned them up and removed a number of them. Also added comments.